tron/core/actionrun.py

```diff
@@ -564,6 +566,8 @@ def start(self, original_command=True) -> Optional[Union[bool, ActionCommand]]:
             self.fail(exitcode.EXIT_INVALID_COMMAND)
             return None
 
+        # Increment the valid actionRuns created
+        prom_metrics.tron_action_runs_valid_counter.labels(executor=str(executor)).inc()
```

Comment:
this isn't necessarily guaranteed to give us the count of valid actionruns: https://github.com/Yelp/Tron/blob/eeb416ec3bf865ac77438c94de3aaa05cd7345bf/tron/core/actionrun.py#L1282 has a couple cases where we won't create a task for various reasons, for example

my suggestion was more along the lines of having the submit_command() function for the various ActionRun subclasses increment this counter rather than doing that here (or just living with the potential mismatch for now by having a single counter that we increment here (without trying to get too deep into whether or not start() actually started a task)